src/main/scala/com/ibm/couchdb/api/builders/QueryStrategy.scala

```diff
@@ -31,9 +31,10 @@ trait QueryStrategy {
   }
 
   def queryByIds[K: W, Q: R](client: Client, db: String, url: String, ids: Seq[K], ps: Map[String, String]): Task[Q] = {
```

Comment:
my settings are at 120 chars, this is just within that limit. But I can restructure this